server/src/main/java/io/crate/exceptions/RoleAlreadyExistsException.java

```diff
@@ -21,27 +21,30 @@
 
 package io.crate.exceptions;
 
-import java.util.Locale;
+import java.io.IOException;
 
-import org.jetbrains.annotations.Nullable;
+import org.elasticsearch.ElasticsearchException;
+import org.elasticsearch.common.io.stream.StreamInput;
+import org.elasticsearch.common.io.stream.StreamOutput;
 
-import io.crate.role.JwtProperties;
 
-public class RoleAlreadyExistsException extends RuntimeException implements ConflictException, UnscopedException {
+public class RoleAlreadyExistsException extends ElasticsearchException implements ConflictException, UnscopedException {
 
-    private RoleAlreadyExistsException(String message) {
+    private final String message;
+
+    public RoleAlreadyExistsException(String message) {
         super(message);
+        this.message = message;
+    }
+
+    public RoleAlreadyExistsException(StreamInput in) throws IOException {
+        super(in);
+        message = in.readString();
```

Comment:
`message` already gets streamed via the base class constructor, see:

https://github.com/crate/crate/blob/a35e98e684999d7155c60ed2abd9af66ec02071a/server/src/main/java/org/elasticsearch/ElasticsearchException.java#L138